### PR TITLE
feat: nft-storage

### PR DESCRIPTION
**Pull request by preschian**
related with #36

### CODE REVIEW

**Comment by yangwao** [comment]
@vikiival

**Comment by vikiival** [comment]
> @vikiival 

It miss the function otherwise i cant replace it on frontend

**Review by vikiival — CHANGES_REQUESTED** [review]
Wont merge until getKey is not implemented

**Review by vikiival — APPROVED** [review]

File: nft-storage-beta/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+app.use('*', logger());
+app.get('/', (c) => c.text('hello nft-storage!'));
+
+app.post('/pinFile', async (c) => {
+  const status = await pinFile(c);
+
+  if (status.cid) {
+    // upload to cf-images in the background
+    c.executionCtx.waitUntil(uploadToCloudflareImages(c, status.cid));
+  }
```
Comment (vikiival) [inline nft-storage-beta/src/index.ts:18]: @preschian maybe check if it's image based on mime-type?

**Pull request closed without merging**